docs/content/guide/external-resources.ngdoc

```diff
@@ -139,7 +139,7 @@ You can find a larger list of AngularJS external libraries at [ngmodules.org](ht
   [CodeAcademy](http://www.codecademy.com/courses/javascript-advanced-en-2hJ3J/0/1),
   [CodeSchool](https://www.codeschool.com/courses/shaping-up-with-angular-js)
 * **Paid online:**
-  [Pluralsight (3 courses)](http://www.pluralsight.com/training/Courses/Find?highlight=true&searchTerm=angularjs),
+  [Pluralsight](https://app.pluralsight.com/library/search?q=angularjs),
```

Comment:
This URL requires the user to sign in. I think we should use https://www.pluralsight.com/search?q=angularjs instead.